front/pages/w/[wId]/builder/data-sources/index.tsx

```diff
@@ -504,14 +504,36 @@ export default function DataSourcesView({
                                           requirements.
                                         </div>
                                       </div>
+
+                                      <div className="flex flex-col gap-y-2">
+                                        <div className="grow text-sm font-medium text-element-800">
+                                          Data Processing
+                                        </div>
+                                        <div className="text-sm font-normal text-element-700">
+                                          By using this service, you hereby
+                                          acknowledge and agree that any data
+                                          contained within the Google Drives,
+                                          and folders you choose to synchronize
+                                          with Dust, will be transmitted to
+                                          third-party entities, including but
+                                          not limited to Artificial Intelligence
+                                          (AI) models, for the purpose of
+                                          processing and analysis. This
+                                          transmission is an integral part of
+                                          the functionality of our service.
+                                          Please note that this process is
+                                          subject to the terms outlined in our
+                                          Privacy Policy and Terms of Service.
+                                        </div>
+                                      </div>
                                       <div className="flex justify-center">
                                         <DropdownMenu.Button>
                                           <Button
                                             variant="secondary"
                                             icon={CloudArrowLeftRightIcon}
                                             disabled={disabled}
                                             onClick={onclick}
-                                            label={label}
+                                            label="Acknowledge and Connect"
```

Comment:
This breaks the feedback showing "Connecting..." no?

Reply:
no because the feedback is actually in the button outside of the modal. The modal disappear right away when you click on this connect button.

Reply:
https://github.com/dust-tt/dust/assets/358965/aeb0d9e0-e967-4b79-af23-6f37f6acb713